specification/appendix/saas_examples/spend_agreements.md

```diff
@@ -0,0 +1,89 @@
+* In the following 2 months, AwesomeCorp has usage at or above the minimum requirement
+* For the final 9 months, AwesomeCorp does not use Acme services
+
+**Example FOCUS dataset:** See an example dataset for Scenario A2 [here](https://docs.google.com/spreadsheets/d/1H69HmngVv-mKpFR-sveY-fSAdR49syqQd7AKOPPQ64A/edit?gid=1740845023#gid=1740845023&range=A25)
+
+Note the following details in the dataset:
+
+* A monthly charge representing the unused minimum monthly amount is charged during months 4 through 11 of the 12-month term
+* The final month has a charge that captures the overall unmet spend requirement for the 12-month contract. Alternatively, this could be provided as two charges, one for the unused portion of the final month, and one to capture the overall unmet spend requirement.
+
+## Scenario B: Prepaid contract
+
+For this scenario B, contract terms include the following terms in addition to the baseline scenario mentioned above:
+
+* The charges will be billed in arrears using monthly invoices
+
+### Scenario B1: Prepaid with no minimum spend requirement per month
+
+Scenario B1 is similar to scenario A1 with the difference being that it's a pre-paid contract.
+
+**Example FOCUS dataset:** See an example dataset for Scenario A1 [here](https://docs.google.com/spreadsheets/d/1H69HmngVv-mKpFR-sveY-fSAdR49syqQd7AKOPPQ64A/edit?gid=1740845023#gid=1740845023&range=A46)
+
+Note the following details in the dataset:
+
+* A purchase record for the initial &dollar;1200 payment is present representing List, Billed, and Contracted cost of the purchase
+* The charge for the unused amount has a &dollar;0 BilledCost (since the total amount was billed with the prepayment). However, the charge captures the unused portion as an EffectiveCost.
+* The unused charge rows apply to the entire Charge Period the contract was signed for.
+* This scenario shows List Cost and Contracted Cost column double counting dynamic (described here in [ListCost](#listcost) and [ContractedCost](#contractedcost)) where either the [ChargeType](#chargetype) Purchase or Usage rows need to be excluded depending on the reporting scenario.
+
+### Scenario B2: Prepaid with a minimum spend requirement per month
+
+Scenario B2 is similar to A2 with the difference being that it's a pre-paid contract.
+
+**Example FOCUS dataset:** See an example dataset for Scenario A2 [here](https://docs.google.com/spreadsheets/d/1H69HmngVv-mKpFR-sveY-fSAdR49syqQd7AKOPPQ64A/edit?gid=1740845023#gid=1740845023&range=A58)
+
+Note the following details in the dataset:
+
+* A purchase record for the initial &dollar;1200 payment is present representing List, Billed, and Contracted cost of the purchase
+* The monthly charge for the unused amount has a &dollar;0 BilledCost (since the total amount was billed with the prepayment). However, the charge captures the unused portion as an EffectiveCost.
```

Comment:
Right now, Datadog is modeling this unused, commitment as an "Adjustment" to help delineate actual usage vs. "adjusted" usage, or the difference to hit predetermined usage goals.  ChargeCategory value, "Adjustment", isn't the best solution IMHO, but it seems better approach than relying on parsing a non-standardized, ChargeDescription string to declaratively know what was actually used vs. what was penalized.